src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -1425,7 +1425,7 @@ public java.sql.ResultSet getProcedureColumns(String catalog, String schema, Str
         // proc name supports escaping
         proc = escapeIDName(proc);
         arguments[0] = proc;
-        arguments[1] = schema;
+        arguments[1] = escapeIDName(schema);
```

Comment:
the comments need updating. They call out proc and col specifically is it implying that the others don't support escaping? This is also making me wondering about catalog does this have the same issue?

Reply:
My suggestion for lines 1426 - 1432:

```suggestion
        // proc, schema, and col names support escaping
        arguments[0] = escapeIDName(proc);
        arguments[1] = escapeIDName(schema);
        arguments[2] = catalog;
        arguments[3] = escapeIDName(col);
```